### ISSUE TITLE
[Bug]: Impossible to uninstall

### ISSUE THREAD

**Issue by esttorhe** [netdata/netdata#15883]
### Bug description

I installed netdata on my raspbian running pihole. My pihole kept dying because of netdata

Today I decided to remove netdata... or so I tried... it doesn't matter how many times I run the uninstaller and make sure it is "gone"
After a while my pihole dies again because of cpu/ram. After rebooting 2 times because it takes forever; netdata is again running...

I can't for the life of me figure out what keeps installing this on my device. I even deleted my account because of this; but my raspberry pi is now unusable and my DNS server keeps crapping on me because of this

### Expected behavior

After running the uninstaller netdata DOESN'T COME BACK 

### Steps to reproduce

1. run `wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --uninstall`
2. Reply yes to all the questions about deleting and removing everything
3. wait until pihole chokes on cpu and ram
4. Reboot the raspberry pi a couple of times
5. Check netdata running happily again and choking the device
...

### Installation method

kickstart.sh

### System info

```shell
/etc/hb-release:v1.0.15
/etc/os-release:PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Raspbian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=raspbian
/etc/os-release:ID_LIKE=debian
```

### Netdata build info

```shell
no idea, just ran the uninstaller at the time of publishing the bug. I'll edit once its back up and running
```

**Comment by esttorhe** [comment]
woke up to this.... again

<img width="942" alt="image" src="https://github.com/netdata/netdata/assets/2642850/9c711366-b7b4-426e-9c74-25695dbe9b79">

**Comment by MrZammler** [comment]
Hi @esttorhe sorry this is affecting you.

One reason for this would be that the auto update feature has not been disabled. Could you check your `/etc/cron.daily` directory if there is a link to `/opt/netdata/usr/libexec/netdata/netdata-updater.sh` ?

I will also check the uninstaller option. Thanks

**Comment by esttorhe** [comment]
Thanks for the reply @MrZammler 

During the uninstaller I get asked if I want to remove the cron entry, I have said yes every time but it sill shows back up.

Let me run the uninstaller once more and then check if the cron entry is still there; to be honest I haven't checked after uninstalling

**Comment by esttorhe** [comment]
![image](https://github.com/netdata/netdata/assets/2642850/5f4d5a2e-79ae-469a-bd49-20e2321e7341)

ok, so it seems things work "ok" until I reboot the raspberry pi... then it is back to when `netdata` was installed... literally everything, even the command history :/

**Comment by MrZammler** [comment]
So, you run the uninstaller, you can see that `/opt/netdata` has been deleted, but then after a restart it re-appears (without installing again manually, or it run via the cron script) ?

Wild guess, but is it possible you're running of a read-only sd card, which loads a basic filesystem on ram? Have you used any special raspberrypi image for it ?

**Comment by esttorhe** [comment]
>Wild guess, but is it possible you're running of a read-only sd card, which loads a basic filesystem on ram? Have you used any special raspberrypi image for it ?

No, I've been running this pi for a long time now (at least a couple of years). Never had similar issues until now that I'm trying to remove `netdata`

**Comment by esttorhe** [comment]
@MrZammler just an FYI; after multiple tests and logs I'm starting to think that my SD might have been corrupted at some point in between me installing `netdata` and me uninstalling it.

I'm 99.9% sure that this was just a bad timing situation which lead me to believe that the tool cannot be uninstalled when in reality nothing is being persisted in the storage medium

I'm going to close the issue; if there's a chance this was in fact a `netdata` issue i will open another ticket

Task for the time looking into it and sorry for the potentially misleading thing

**Comment by MrZammler** [comment]
@esttorhe No worries! Thanks for the follow up, do let us know if indeed there is an issue.
